crates/matrix-sdk-base/src/store/indexeddb_store.rs

```diff
@@ -0,0 +1,1043 @@
+// Copyright 2021 The Matrix.org Foundation C.I.C.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[allow(unused_imports)]
```

Comment:
The allow isn't needed, no? Actually the imports shouldn't be needed either since they are nowadays part of the prelude[[1]].

[1]: https://doc.rust-lang.org/edition-guide/rust-2021/prelude.html